chatgpt-shell-openai.el

```diff
@@ -91,18 +104,23 @@ HANDLER, FILTER and OTHER-PARAMS."
          :token-width 3
          ;; https://platform.openai.com/docs/models/o1
          :context-window 200000
+         :reasoning-effort chatgpt-shell-openai-default-reasoning-effort
          :validate-command #'chatgpt-shell-validate-no-system-prompt)
         (chatgpt-shell-openai-make-model
          :version "o1-preview"
          :token-width 3
          ;; https://platform.openai.com/docs/models/gpt-01
          :context-window 128000
+         ;; Reasoning effort is only supported for o1-pro, o1 and o3-mini.
+         ;; :reasoning-effort chatgpt-shell-openai-default-reasoning-effort
```

Comment:
Is this meant to be commented out? If so, perhaps consider `:reasoning-effort nil`

Reply:
It is meant to be commented out. :reasoning-effort is already nil by default when not provided. I wanted a comment here so someone wouldn't think it should be added in the future.

Reply:
I prefer the commented code but this is your project so I won't argue 🙂